StageChangeTest.cls

```diff
@@ -0,0 +1,32 @@
+@IsTest public class StageChangeTest {

```

Comment:
should have a test when we update the test and StageName will not be CloseWon and CloseLost